web/tests/upload.test.js

```diff
@@ -84,13 +88,22 @@ test("get_item", () => {
         $(`#edit_form_${CSS.escape(75)} .upload_banner.file_id_60`),
     );
 
+    $(`#edit_form_${CSS.escape(2)} .upload_banner`).set_find_results(
+        ".upload_banner_cancel_button",
+        $(".upload_banner_cancel_button"),
+    );
+    assert.equal(
+        upload.get_item("upload_banner_cancel_button", {mode: "edit", row: 2}, "id_34"),
+        $(`#edit_form_${CSS.escape(2)} .upload_banner.file_id_34 .upload_banner_cancel_button`),
+    );
+
     $(`#edit_form_${CSS.escape(2)} .upload_banner`).set_find_results(
         ".main-view-banner-close-button",
         $(".main-view-banner-close-button"),
     );
     assert.equal(
-        upload.get_item("upload_banner_close_button", {mode: "edit", row: 2}, "id_34"),
-        $(`#edit_form_${CSS.escape(2)} .upload_banner.file_id_34 .main-view-banner-close-button`),
+        upload.get_item("upload_banner_hide_button", {mode: "edit", row: 2}, "id_34"),
+        $(`#compose_banners .upload_banner.file_id_34 .main-view-banner-close-button`),
     );
```

Comment:
@yogesh-sirsat ,I have replaced the name `upload_banner_close_button` with `upload_banner_hide_button` as it seems more appropriate to what we are trying to do. I have made the necessary changes and updated the commit description accordingly. Could you please review the PR? Thank you!